src/Backend/Core/Js/Components/Meta.js

```diff
@@ -88,6 +88,10 @@ export class Meta {
               url = data.data
               $(options.urlSelector).val(url)
               $(options.generatedUrlSelector).html(url)
+
+              if (url.trim() === 'home') {
+                $(options.generatedUrlSelector).val('')
```

Comment:
Shouldn't you just change the value of URL tot empty string before the lines 89 and 90?

Reply:
Also, wouldn't this one give issues if I want to name a different page home?
You should do this based on the page id

Reply:
fixed d060df6dc12c31f525250398e1d0d541a030a923